mp/src/game/server/momentum/mom_replay_system.cpp

```diff
@@ -144,20 +144,20 @@ void CMomentumReplaySystem::FinishRecording()
     SetReplayHeaderAndStats();
 
     char newRecordingPath[MAX_PATH];
-    if (StoreReplay(newRecordingPath, MAX_PATH))
-    {
-        Log("Recording Stopped! Ticks: %i\n", m_pRecordingReplay->GetFrameCount());
+    bool bStoredReplay = StoreReplay(newRecordingPath, MAX_PATH);
 
-        const auto pReplaySavedEvent = gameeventmanager->CreateEvent("replay_save");
-        if (pReplaySavedEvent)
-        {
-            pReplaySavedEvent->SetBool("save", true);
-            // replaySavedEvent->SetString("filename", newRecordingName);
-            pReplaySavedEvent->SetString("filepath", newRecordingPath);
-            pReplaySavedEvent->SetInt("time", static_cast<int>(m_pRecordingReplay->GetRunTime() * 1000.0f));
-            gameeventmanager->FireEvent(pReplaySavedEvent);
-        }
+    const auto pReplaySavedEvent = gameeventmanager->CreateEvent("replay_save");
+    if (pReplaySavedEvent)
+    {
+        pReplaySavedEvent->SetBool("save", bStoredReplay);
+        pReplaySavedEvent->SetString("filepath", newRecordingPath);
+        pReplaySavedEvent->SetInt("time", static_cast<int>(m_pRecordingReplay->GetRunTime() * 1000.0f));
+        gameeventmanager->FireEvent(pReplaySavedEvent);
+    }
 
+    if (bStoredReplay)
+    {
+        Log("Recording Stopped! Ticks: %i\n", m_pRecordingReplay->GetFrameCount());
```

Comment:
While we're here can we implement the simple suggestion of also printing out the run time to console? Should just be a call to `MomUtil::FormatTime` with the recording replay's *run time* (not frame count)